lib/l10n/app_en.arb

```diff
@@ -3,5 +3,17 @@
   "email": "Email",
   "password": "Password",
   "login": "Login",
-  "forgot_password": "Forgot?"
+  "forgot_password": "Forgot?",
+  "today": "Today",
+  "yesterday": "Yesterday",
+  "days_ago": "{daysAgo} days ago",
```

Comment:
Does flutter have a plural string feature for this? 🤔  If the `daysAgo` is `1`, it can change to `1 day ago`.
Maybe we can use this => https://pub.dev/packages/intl_extension
Ref : https://stackoverflow.com/a/59845992/11393626